docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -12,15 +12,33 @@ videoBanner: UFhT52d5bYg
 - AWS account with a Redshift cluster and permissions to create and attach IAM
   policies.
 
-## Step 1/5. Install Teleport
+## Step 1/6. Install Teleport
 
 (!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-## Step 2/5. Create Teleport user
+## Step 2/6. Create Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 3/5. Configure IAM
+## Step 3/6. Create database agent configuration
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Create the Database Service configuration file:
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:3080 \
+   --token=/tmp/token \
+   --redshift-discovery=us-west-1
+```
+
+The command will generate a Database Service configuration with Redshift
+database auto-discovery enabled on the `us-west-1` region and place it at the
+`/etc/teleport.yaml` location.
+
+## Step 4/6. Configure IAM
 
 ### Create an IAM policy for Teleport
```

Comment:
Since there's only one H3 in this H2 section, should we make "Create an IAM policy for Teleport" the title of this H2 (instead of "Configure IAM") and remove this H3 heading?